monitoring/osd-managing-metrics.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-metrics"]
+= Managing metrics
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: managing-metrics
+
+toc::[]
+
+This section contains examples that show how to manage your metrics.
+
+// Understanding metrics
+include::modules/osd-monitoring-understanding-metrics.adoc[leveloffset=+1]
+
+// Setting up metrics collection for user-defined projects
+include::modules/osd-monitoring-setting-up-metrics-collection-for-user-defined-projects.adoc[leveloffset=+1]
+include::modules/osd-monitoring-deploying-a-sample-service.adoc[leveloffset=+2]
+include::modules/osd-monitoring-specifying-how-a-service-is-monitored.adoc[leveloffset=+2]
+
+* xref:../monitoring/accessing-monitoring-for-user-defined-projects.adoc[Accessing monitoring for user-defined projects]
```

Comment:
link is wrong
should be
../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc

Reply:
Fixed.